server/svix-server/src/core/message_app.rs

```diff
@@ -157,6 +157,23 @@ pub struct CreateMessageEndpoint {
     pub deleted: bool,
 }
 
+impl CreateMessageEndpoint {
+    pub fn get_valid_signing_keys(&self) -> Vec<&EndpointSecretInternal> {
```

Comment:
```suggestion
    pub fn valid_signing_keys(&self) -> Vec<&EndpointSecretInternal> {
```
Getters in rust don't have the word get.